### DISCUSSION TITLE
Future of standalone chart

### DISCUSSION THREAD

**Discussion by tbnguyen1407** [redpanda-data/connect#2634] [General]
Hello, current "Benthos" chart is here

https://github.com/benthosdev/benthos-helm-chart

Is there equivalent chart for standalone "Connect"?

The new README forces using redpanda/rpk image to deploy "Connect" but I am wondering if it is still possible for a standalone "Connect" chart without all the bloats.

**Notes**
- Currently the standalone "Connect" image is `ghcr.io/redpanda-data/connect`, which is not documented in the README, which probably still works with "Benthos" chart

**Comment by Jeffail** [comment]
Hey @tbnguyen1407, our intention is to bring it over, and yes ideally that would mean using the standalone connect image. We're still working out some details around how maintainence is going to work.
